packages/store/addon/-private/system/fetch-manager.ts

```diff
@@ -451,7 +464,9 @@ export default class FetchManager {
       // will once again convert the records to snapshots for adapter.findMany()
       let snapshots = new Array<Snapshot>(totalItems);
       for (let i = 0; i < totalItems; i++) {
-        let options = optionsMap.get(identifiers[i]);
+        // we know options is in the map due to having just set it above
+        // but TS doesn't know so we cast it
+        let options = optionsMap.get(identifiers[i]) as Dict<any>;
```

Comment:
Suggestion: use the same type as in the declaration, `as Object`, or use `!`. (I don't love `!` and it's easy for it to get missed when fixing these kinds of things, so `as Object` is probably better.)

Reply:
made them both `Dict<unknown>`